index.js

```diff
@@ -738,6 +738,21 @@
   //. 'Just([1, 2, 3])'
   //. ```
 
+  //# Maybe#inspect :: Maybe a ~> String
+  //.
+  //. Returns the string representation of the Maybe. This function is
+  //. identical to the `toString` method. It is used by `util.inspect` and the
+  //. REPL to format a Maybe for display.
```

Comment:
This is not *identical* to the `toString` method. We could remove the second sentence and add a second paragraph:

```javascript
  //. See also [`Maybe#toString`](#Maybe.prototype.toString).
```